python/whylogs/experimental/extras/confighub.py

```diff
@@ -0,0 +1,130 @@
+import logging
+import os
+from abc import ABC, abstractmethod
+from typing import List, Optional
+
+from semantic_version import Version
+
+logger = logging.getLogger(__name__)
+
+
```

Comment:
This API might not be great if we expect a lot of concurrent editing. Is that a use case we need to support?

Reply:
I think that's probably ok for now